lib/resolvers/parse/ast2cqn/where.js

```diff
@@ -30,13 +31,25 @@ const _to_xpr = (ref, gqlOperator, value) => {
 const _objectFieldTo_xpr = (objectField, columnName) => {
   const ref = { ref: [columnName] }
   const gqlOperator = objectField.name.value
+  const operand = objectField.value
+
+  if (gqlOperator === LOGICAL_OPERATORS.in) {
+    return [
+      ref,
+      _gqlOperatorToCdsOperator(gqlOperator),
+      {
+        list:
+          operand.kind === Kind.LIST ? operand.values.map(value => ({ val: value.value })) : [{ val: operand.value }]
+      }
+    ]
```

Comment:
Imo slightly more readable. What do you think?
```suggestion
    const list = operand.kind === Kind.LIST ? operand.values.map(value => ({ val: value.value })) : [{ val: operand.value }]
    return [ref, _gqlOperatorToCdsOperator(gqlOperator), { list }]
```